### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Altlama**
Please update so it is easy for everyone  tks

### CODE REVIEW

File: community/passbolt/1.2.12/README.md [hunk 1]
```diff
@@ -13,10 +13,14 @@
...
+CD ..  several time to move to the level you see 
+![Captura de tela 2024-01-09 101333](https://github.com/user-attachments/assets/f981750f-5fb9-405c-a1c2-73130436d058)
+
+and run the following command replacing the
 values (`user@example.com`, `first_name`, `last_name`) with your own values.
 
 ```shell
-/usr/share/php/passbolt/bin/cake passbolt register_user -r admin \
-  -u user@example.com -f first_name -l last_name
+/usr/share/php/passbolt/bin/cake passbolt register_user -r admin -u user@example.com -f first_name -l last_name
```
Comment (stavros-k) [inline community/passbolt/1.2.12/README.md:24]:
```suggestion
/usr/share/php/passbolt/bin/cake /passbolt register_user -r admin -u user@example.com -f first_name -l last_name
```

File: community/passbolt/1.2.12/README.md [hunk 2]
```diff
@@ -13,10 +13,14 @@
 
 ## Register admin user
 
-Connect to the container's shell and run the following command replacing the
+Connect to the container's shell
+CD ..  several time to move to the level you see 
+![Captura de tela 2024-01-09 101333](https://github.com/user-attachments/assets/f981750f-5fb9-405c-a1c2-73130436d058)
+
```
Comment (stavros-k) [inline community/passbolt/1.2.12/README.md:19]:
```suggestion
```

**Review by stavros-k — CHANGES_REQUESTED** [review]
There is no need to do all this `cd ..`.
a `/` before the passbolt will solve the issue.

Please also revert any changes you did on this file and apply (along with my suggestions) in the `library/ix-dev/community/passbolt/README.md` and app-readme.md files.

Thanks

**Review by Altlama — COMMENTED** [review]
looks good now?

**Comment by stavros-k** [comment]
> looks good now?

Please only update these files:
`library/ix-dev/community/passbolt/README.md`
`library/ix-dev/community/passbolt/app-readme.md`

Thanks

**Review by Altlama — COMMENTED** [review]
looks good

**Review by Altlama — COMMENTED** [review]
looks good

**Comment by Altlama** [comment]
Hello
The proposed solution for the directory  give the bellow error
$ /usr/share/php/passbolt/bin/cake /passbolt register_user -r admin -u ***@****.com -f first_name -l last_name

_**Unknown command `cake /passbolt`. Run `cake --help` to get the list of commands.
Did you mean: `passbolt`?

Other valid choices:

- version
- console
- passbolt**_

**Comment by stavros-k** [comment]
I just tried the existing command in the docs (without the `/` and without doing `cd /`) and worked fine.
I'm not sure what issue you had that made you do all these extra steps.

See:


```
$  /usr/share/php/passbolt/bin/cake passbolt register_user -r admin -u s@s.com -f first_name -l last_name

     ____                  __          ____  
    / __ \____  _____ ____/ /_  ____  / / /_ 
   / /_/ / __ `/ ___/ ___/ __ \/ __ \/ / __/ 
  / ____/ /_/ (__  |__  ) /_/ / /_/ / / /    
 /_/    \__,_/____/____/_.___/\____/_/\__/   

 Open source password manager for teams
-------------------------------------------------------------------------------
User saved successfully.
To start registration follow the link provided in your mailbox or here: 
http://10.20.30.230:30097/setup/start/713f1feb-7a7b-4ced-b264-ba2e278f386f/6c954a8e-e59d-4cb7-a831-c23f18460089
$ pwd
/usr/share/php/passbolt
$ 
```

**Comment by Altlama** [comment]
https://www.truenas.com/community/threads/passbolt-cant-change-initial-user.114501/
i have the same issue like in that post   i'm evaluating passbolt so i did redo the install and if i don't cd to that directory it is not working.  I run truenas scale Dragonfish-24.04.2.2

**Comment by stavros-k** [comment]
Can you please show a step by step (with video or pictures) what you do and what you see?
Because the forum post, most of the issues is that the wrong pod was selected.

**Comment by Altlama** [comment]
Sure  when i'm back next week
tks.

**Comment by Altlama** [comment]
Hello ,
I did remove all apps, K8s , IX app dataset and other apps dataset. Did re install and no issue that time. As scale is moving from kubernetes to Docker next release i will not involve myself more for now. Thank you for the support.

**Pull request closed without merging**